File: search_insert_position.py
class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:
        try:
            return nums.index(target)
        except ValueError:
            index = 0
            for numbers in range(len(nums)):
                if target>=nums[numbers]:
                    index = numbers + 1
            return index

File: search_insert_position.c
int searchInsert(int* nums, int numsSize, int target)
{
    int value_idx = 0;
    int low_idx = 0;
    int high_idx = numsSize - 1;

    while(low_idx<=high_idx)
    {
        int mid = (int)(low_idx + high_idx)/2;
        if(target == nums[mid])
        {   
            value_idx = mid;
            
            return value_idx;
        }
        else if (target > nums[mid])
        {
            low_idx = mid + 1;
        }
        else if (target < nums[mid])
        {
            high_idx = mid - 1;
        }

    }

    return low_idx;
}